BALSAMIC/utils/constants.py

```diff
@@ -619,7 +619,8 @@
     "strelka": "varcall_py27",
     "manta": "varcall_py27",
     "cnvkit": "varcall_cnvkit",
-    "delly": "varcall_delly"
+    "delly": "varcall_delly",
+    "ascat": "ascatngs"
 }
```

Comment:
As we discussed `BIOINFO_TOOL_SUBMODULE` should be created here.

```suggestion
}

BIOINFO_TOOL_SUBMODULE = {
    "ascat": "ascatngs"
}
```

And in https://github.com/Clinical-Genomics/BALSAMIC/blob/5b754baf1b2d151dc1fcd25d35b117d36f4bdbeb/BALSAMIC/commands/config/case.py#L12-L13 add `BIOINFO_TOOL_SUBMODULE` here as well.

And merge two dictionaries here:

https://github.com/Clinical-Genomics/BALSAMIC/blob/5b754baf1b2d151dc1fcd25d35b117d36f4bdbeb/BALSAMIC/commands/config/case.py#L134 merge two dictionaries `BIOINFO_TOOL_SUBMODULE` and `BIOINFO_TOOL_ENV`.

Ultimately in the new rule, it'll pickup correct ascatngs container.